Demo/src/main/java/com/braintreepayments/demo/MainActivity.java

```diff
@@ -192,6 +192,10 @@ protected void reset() {
 
     @Override
     protected void onAuthorizationFetched() {
+        if (dropInClient != null) {
+            return;
+        }
```

Comment:
I'm a little confused why we need this?

Reply:
We can get rid of this actually. There was an infinite `onResume` loop because of storage permissions that was happening. We added `WRITE_EXTERNAL_STORAGE` to the demo manifest now so this should only be called once.